content/study-configuration/recruitment.md

```diff
@@ -8,10 +8,49 @@ The "recruitment" configuration describes how/where recruitment will take place.
 
 There are currently 3 types of recruitments:
 
-1. Simple.
+1. [Simple](#simple)
+2. [Pipeline Experiment](#pipeline-experiment)
+3. [Destination Experiment](#destination-experiment)
 
+## Simple
 
-2. Destination Experiment. Use this when you want to create a multi-arm randomized experiment (A/B test on Facebook) where some of your sample is sent to different "destinations".
+This is the easiest of the recruiting strategies where the Virtual Labs platform will
+routinely check your ad campaign and update the campaign accordingly to try and
+achieve your recruiting goals.
 
-3. Pipeline Experiment. In this design, we generate an A/B test on Facebook but instead of sending your sample to different destinations, we run the ads at different times (a pipeline experiment design). You can set up how long each arm runs, and how long they are offset from the start of the previous arm.
+You will need to provide the following fields:
 
+- `Ad Campaign Name`: This is the name of the ad campaign that this recruitment
+    should be targeting
+- `Budget`: The amount you would like to spend in order to recruit your target
+- `Max Sample`: This is the maximum sample size
+- `Start Date`: The date in which the recruiting should start
+- `End Date`: The date on which the recruiting should end
+
+## Destination Experiment 
+
+Use this when you want to create a multi-arm randomized experiment (A/B test on Facebook) where some of your sample is sent to different [destinations]({{< ref destination.md >}} "Destination Configuration"). You will need to have the destinations configured and you will then need to set the following fields:
+
+- `Ad Campaign Name Base`:
```

Comment:
@nandanrao could you give a brief explanation of what a `Ad Campaign Name Base` is I understand the high level of it but cant really give a good explanation of it

Reply:
`Ad Campaign Name Base` - the name of the ad campaign(s) that vlab will create and use for recruitment. If you have more than one campaign (i.e. Destination Experiment), it will append `-1`, `-2`, etc. to this name.

Reply:
Awesome! Thank you!